Mentoring  Program/Abstract/IStoredProceduresAndUDFHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MentoringProgram.DataModels;
+using System.Collections.Generic;
+
+namespace MentoringProgram.Abstract
+{
+    public interface IStoredProceduresAndUDFHelper
```

Comment:
I'd put similar methods from two interfaces into single IGameRepository. If you need to use both implementations simultaneously, you can register both in IoC container and use IEnumerable<IGameRepository> in client class constructor.